software/conduwuit.yml

```diff
@@ -0,0 +1,10 @@
+name: conduwuit
+website_url: https://conduwuit.puppyirl.gay
+description: conduwuit is a well-maintained, featureful, hard-fork of Conduit, which is a Matrix chat homeserver.
```

Comment:
```suggestion
description: Well-maintained, featureful Matrix chat homeserver (fork of Conduit).
```

https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines

Reply:
https://conduwuit.puppyirl.gay/differences.html @nodiscc

Reply:
Also applied the suggested description.

Reply:
https://conduwuit.puppyirl.gay/differences.html is more than fine, thank you, I did not see it at first.